src/main/java/com/signavio/examples/dmn/testsuite/TestSuiteReader.java

```diff
@@ -2,34 +2,22 @@
 
 import java.io.IOException;
 import java.io.InputStream;
-import java.io.InputStreamReader;
 
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
-import com.google.gson.JsonObject;
-import com.google.gson.stream.JsonReader;
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.signavio.bdm.testlab.exchange.TestSuite;
-import org.json.JSONObject;
 
 public class TestSuiteReader {
 	
-	private static final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
-	
-	
 	public TestSuite parse(InputStream testSuite) {
 		try {
-			JsonReader reader = createTestSuiteReader(testSuite);
-			JsonObject testSuiteJson = GSON.fromJson(reader, JsonObject.class);
-			
-			return TestSuite.fromJson(new JSONObject(testSuiteJson.toString()));
+			return new ObjectMapper()
+					.setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY)
+					.readValue(testSuite, TestSuite.class);
```

Comment:
now that `new JSONObject()` is gone here, I think we can remove the `org.json` dependency as well. At least i didn't find any more usages :shrug:

Reply:
Tried it out but unfortunately, Jackson needs it:

	at com.fasterxml.jackson.databind.deser.DeserializerCache.findValueDeserializer(DeserializerCache.java:142)
	at com.fasterxml.jackson.databind.DeserializationContext.findRootValueDeserializer(DeserializationContext.java:491)
	at com.fasterxml.jackson.databind.ObjectMapper._findRootDeserializer(ObjectMapper.java:4711)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4520)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3503)
	at com.signavio.examples.testsuite.TestSuiteReader.parse(TestSuiteReader.java:17)
	at com.signavio.examples.testsuite.TestSuiteUtil.readTestSuite(TestSuiteUtil.java:35)
	at com.signavio.examples.dmn.DmnWithTestCasesExample.execute(DmnWithTestCasesExample.java:45)
	at com.signavio.examples.SignavioExamples.executeExample(SignavioExamples.java:44)
	at com.signavio.examples.SignavioExamples.executeDmnExamples(SignavioExamples.java:26)
	at com.signavio.examples.SignavioExamples.main(SignavioExamples.java:18)
Caused by: java.lang.ClassNotFoundException: org.json.JSONException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)

Reply:
Damn. Alright then :+1: